index.html

```diff
@@ -13,23 +13,202 @@
-				<li><a href="#"><img src="./images/medium.svg" alt="Medium Logo"></a></li>
-				<li><a href="#"><img src="./images/github.svg" alt="Github Logo"></a></li>
-				<li><a href="#"><img src="./images/angellist.svg" alt="Angellist Logo"></a></li>
-			</ul>
-		</main>
+
+		<div class="app-items">
+			<nav class="app-nav">
+				<a id="logo" href="#">My logo</a>
+				<i id="menu-icon" class="bi bi-list"></i>
+			</nav>
+			<main class="app-greeting">
+				<h1>I'm Sane great to see you!</h1>
+				<p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work
+					and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+				<p id="lets-connect">LET'S CONNECT</p>
+				<ul class="socials">
+					<li><a href="#"><img src="./images/twitter.svg" alt="Twitter logo"></a></li>
+					<li><a href="#"><img src="./images/linkedin.svg" alt="Linkedin Logo"></a></li>
+					<li><a href="#"><img src="./images/medium.svg" alt="Medium Logo"></a></li>
+					<li><a href="#"><img src="./images/github.svg" alt="Github Logo"></a></li>
+					<li><a href="#"><img src="./images/angellist.svg" alt="Angellist Logo"></a></li>
+				</ul>
+			</main>
+		</div>
+
+		<div class="projects">
+			<div class="project-images">
+				<img src="./images/tonic1-img.svg" alt="Picture of project">
+			</div>
+			<main class="project-title">
+				<h2>Tonic</h2>
+				<ul class="list">
+					<li id="canopy">Canopy</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">Back End Dev</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">2015</li>
+				</ul>
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+				</p>
+				<ul class="program-languages">
+					<li><a id="language-name" href="#">html</a></li>
+					<li><a id="language-name" href="#">css</a></li>
+					<li><a id="language-name" href="#">javascript</a></li>
+				</ul>
+				<div><a class="see-projects" href="#">See Projects</a></div>
+			</main>
+		</div>
+
+		<div class="projects">
+			<div class="project-images">
+				<img src="./images/availability.svg" alt="Picture of project">
+			</div>
+			<main class="project-title">
+				<h2>Multi-Post<br> Stories</h2>
+				<ul class="list">
+					<li id="canopy">Canopy</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">Back End Dev</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">2015</li>
+				</ul>
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+				</p>
+				<ul class="program-languages">
+					<li><a id="language-name" href="#">html</a></li>
+					<li><a id="language-name" href="#">css</a></li>
+					<li><a id="language-name" href="#">javascript</a></li>
+				</ul>
+				<div><a class="see-projects" href="#">See Projects</a></div>
```

Comment:
- [ ] Please make sure all your buttons follow the button styling: Active, Hover, and Pressed as defined on the design :+1: 
![Screenshot from 2022-09-28 11-10-48](https://user-images.githubusercontent.com/6615394/192765097-5c53f9b1-3c37-4f27-9544-ff2f8c5e52ce.png)